index.js

```diff
@@ -2,36 +2,41 @@
 
 var isObj = require('isobj');
 
-var Flatten = module.exports = function () {
-  var blacklist = Array.prototype.slice.call(arguments);
+var Flatten = module.exports = function (obj, options) {
+  options = options || {};
+  options.blacklist = options.blacklist || [];
```

Comment:
I don't think it's a good idea to mutate the state of the `options` object (if I pass in an object it will change for me as the caller as well). Maybe just store the two options as local variables:

```js
var blacklist = options.blacklist || [];
var separator = options.separator == null ? '.' : options.separator
```